deployments/mainnet/weth/migrations/1666906000_configurate_seed_reserves_and_rewards.ts

```diff
@@ -66,19 +66,29 @@ export default migration('1666906000_configurate_seed_reserves_and_rewards', {
         args: [comet.address, COMPAddress],
       },
 
-      // 6. Wrap ETH as WETH and send from Timelock to Comet to seed reserves
+      // 6. Wrap some ETH as WETH
       {
         contract: WETH,
         signature: "deposit()",
         args: [],
-        value: exp(1_000, 18)
-      }, {
+        value: 358052565869157684316n, // 500e18 - current balance
```

Comment:
Replace with `500e18 - WETH.balanceOf(timelock.address)`?

Reply:
It's tied to the 500 number though, and the checks we do

Reply:
Actually nevermind 500 - could be good? But idk I think this is actually good/fine and checks are almost done